MsbuildDependencyVisualizer.UI/MainWindow.xaml.cs

```diff
@@ -112,73 +132,61 @@ private void OnBrowseClick(object sender, RoutedEventArgs e)
             }
 
             var controller = await this.ShowProgressAsync("Please wait...", "Processing files..");
-
             try
             {
                 await TraverseProjects(txtPath.Text, true);
-                _graph = new Graph();
-
                 double i = 1.0;
+                var _graph = new Graph();
                 foreach (KeyValuePair<string, List<string>> item in _dependencies)
                 {
                     double percent = i / _dependencies.Count;
                     controller.SetProgress(percent);
-
-                    controller.SetMessage($"Processing... {item.Key}");
+                    controller.SetMessage(string.Format("Processing... {0}", item.Key));
```

Comment:
Why do you want to remove string interpolation? Its a C# 6 feature and I would like it to remain.